data/Templates/FhirToHL7v2/Segments/_PID.liquid

```diff
@@ -46,7 +48,7 @@
 {% comment %}  0 Message Type                          {% endcomment %}PID|
 {% comment %}  1 Set ID - PID                          {% endcomment %}1|{% comment %} PID.1 does not get sent to Fhir {% endcomment %}
 {% comment %}  2 Patient ID                            {% endcomment %}{% comment %}1234567^4^M11^test^MR^University Hospital^19241011^19241012{% endcomment %}|
-{% comment %}  3 Patient Identifier List               {% endcomment %}{% comment %}PATID1234^5^M11^test1&2.16.1&HCD^MR^GOOD HEALTH HOSPITAL~123456789^^^USSSA^SS{% endcomment %}|
```

Comment:
As best I can tell, `GOOD HEALTH HOSPITAL` does not exist in the FHIR. I think it's helpful for reviewers if you can call this information out in the PR somewhere.